scripts/fetch/fetch_utils.R

```diff
@@ -0,0 +1,62 @@
+# Fetch utility functions
+
+#' Convert a vector of counties into sf polygons
+#' 
+#' counties must be strings of "[state]:[county]". See ?maps::county.fips
+#' 
+county_to_sf <- function(counties, crs = 3086){
+  library(sf)
+  library(maps)
+  
+  sf_poly <- sf::st_as_sf(map("county", region = counties, plot = FALSE, fill=TRUE))
+  sf_poly_transf <- st_transform(sf_poly, crs)
+  
+  return(sf_poly_transf)
+}
+
+#' Convert an sf object into an sp object
+#' 
+sf_to_sp <- function(sf_poly, crs = "+proj=longlat +datum=WGS84 +ellps=WGS84 +towgs84=0,0,0"){
```

Comment:
is this function used?

Reply:
I wouldn't reproject by default in a basic type conversion function.

Reply:
Yes, it's used because I couldn't get geoknife to work with `sf`, so I converted to `sp`. We might be able to implement that but this was my current solution.

Where should reprojection happen?

Reply:
I'm going to move transformations to the actual fetchers instead of the helper functions

Reply:
I think the best bet would be to move the reproduction to where we create or fetch the data. That was you get it set right at the beggining and don't mess with it again.